### ISSUE TITLE
Rule to configure/disable async/await behavior

### ISSUE THREAD

**Issue by danyf90** [nicklockwood/SwiftFormat#861]
Hello,
I have a problem that I think has been introduced in version 0.47.8 as part of `Added preliminary support for upcoming async/await syntax`.

We are using [a library](https://github.com/malcommac/Hydra) that defines a function named `await` (the same as the Swift keyword) and when formatting using a recent version of SwiftFormat, it is adding a space between the `await` keyword and the function call parenthesis.
For example, `await(promise)` becomes `await (promise)`, which I don't think is intended in this case.

Could you please confirm whether this behavior is expected and if there is a way to disable/configure it?

Thanks 🙌

**Comment by nicklockwood** [comment]
@danyf90 this is not expected, I'll try to fix it asap. A workaround for now would be to disable the `spaceAroundParens` rule, either by adding `--disable spaceAroundParens` to the SwiftFormat config, or by adding the following comment to the top of any affected file:

```swift
// swiftformat:disable spaceAroundParens
```

**Comment by danyf90** [comment]
Ciao @nicklockwood, sorry for pushing on this.
Did you had any time to have a look at the issue? 🙏

**Comment by nicklockwood** [comment]
Sorry, I've been busy with other things but I'll try to take a look asap.

**Comment by nicklockwood** [comment]
@danyf90 I've pushed a workaround to the [develop](https://github.com/nicklockwood/SwiftFormat/tree/develop) branch if you'd like to try it out (you'll have to build SwiftFormat yourself from source).

The workaround is that the space won't be added after `await` if you set `--swiftversion` to something less than `5.5`. This may need to change in the final release as I'm not sure if async/await will actually ship in 5.5 or some later Swift version.

By the way, I tried building the Hydra framework with the latest swift toolchain (the one that includes async/await) and it doesn't build, so I suspect that the Hydra async method will have to be renamed anyway, at which point this won't be an an issue.
